packages/components/src/components/table/table-cell/table-cell.tsx

```diff
@@ -17,13 +18,16 @@ export class TableCell {
   /** Displays slotted text multiline or forced into a single line. */
   @Prop() public multiline?: boolean = false;
 
+  /** Adapts the color when used on dark background. */
+  @Prop() public theme?: Theme = 'light';
```

Comment:
Developer experience is kind of bad when one has to set the theme prop on every single child. That would be way nicer if the parent `p-table` could take care of it.

Reply:
Fixed it